stix_shifter_modules/cybereason/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,2299 @@
+      "object": "file"
+    },
+    "specialBuild": {
+      "key": "file.extensions.x-cybereason-file.special_build",
+      "object": "file"
+    },
+    "internalName": {
+      "key": "file.extensions.x-cybereason-file.internal_name",
+      "object": "file"
+    },
+    "comments": {
+      "key": "file.extensions.x-cybereason-file.comments",
+      "object": "file"
+    },
+    "applicationIdentifier": {
+      "key": "file.extensions.x-cybereason-file.application_identifier",
+      "object": "file"
+    }
+  },
+  "User": {
+    "username": [
+      {
+        "key": "user-account.user_id",
+        "object": "user"
+      },
+      {
+        "key": "x-oca-asset.user_ref",
+        "object": "asset",
+        "references": "user"
+      }
+    ],
+    "adLogonName": [
+      {
+        "key": "email-addr.value",
+        "object": "user_email"
+      },
+      {
+        "key": "email-addr.belongs_to_ref",
+        "object": "user_email",
+        "references": "user"
+      }
+    ],
+    "isAdmin": {
+      "key": "user-account.extensions.x-cybereason-user.is_admin",
+      "object": "user"
+    },
+    "adCreated": [
+      {
+        "key": "user-account.account_created",
+        "object": "user",
+        "transformer": "EpochToTimestamp"
+      },
+      {
+        "key": "first_observed",
+        "cybox": false,
+        "transformer": "EpochToTimestamp"
+      },
+      {
+        "key": "last_observed",
+        "cybox": false,
+        "transformer": "EpochToTimestamp"
+      }
+    ],
+    "domain": {
+      "key": "user-account.extensions.x-cybereason-user.domain",
+      "object": "user"
+    },
+    "ownerOrganization.name": {
+      "key": "user-account.extensions.x-cybereason-user.organizational_unit",
+      "object": "user"
+    },
+    "ownerMachine": {
+      "key": "user-account.extensions.x-cybereason-user.last_logged_machine",
+      "object": "user"
+    },
+    "isLocalSystem": {
+      "key": "user-account.extensions.x-cybereason-user.is_local_system",
+      "object": "user"
+    },
+    "adDepartment": {
+      "key": "user-account.extensions.x-cybereason-user.department",
+      "object": "user"
+    },
+    "adCountry": {
+      "key": "user-account.extensions.x-cybereason-user.company",
+      "object": "user"
+    },
+    "adCompany": {
+      "key": "user-account.extensions.x-cybereason-user.company",
+      "object": "user"
+    },
+    "adAssociatedDomain": [
+      {
+        "key": "domain-name.value",
+        "object": "user_domain"
+      },
+      {
+        "key": "user-account.extensions.x-cybereason-user.associated_domain",
+        "object": "user"
+      },
+      {
+        "key": "user-account.extensions.x-cybereason-user.associated_domain",
+        "object": "user",
+        "references": "user_domain"
+      }
+    ],
+    "emailAddress": [
+      {
+        "key": "user-account.extensions.x-cybereason-user.email_address",
```

Comment:
This should be an `email-addr` object instead.

Reply:
Thanks for comment. We will update the from_stix and to_stix mappings as per your suggestions.

Reply:
We have pushed in all the identified fields .